instrumentation/spring/spring-kafka-2.7/library/build.gradle.kts

```diff
@@ -10,7 +10,8 @@ dependencies {
 
   implementation(project(":instrumentation:kafka:kafka-clients:kafka-clients-common:library"))
 
-  compileOnly("org.springframework.kafka:spring-kafka:2.7.0")
+  // compiling against 2.8.0 to use methods that are not present in 2.7
+  compileOnly("org.springframework.kafka:spring-kafka:2.8.0")
```

Comment:
If bumping to v2.8.0, does the folder and package naming need to be updated too? e.g.:
* spring-kafka-2.7/ -> spring-kafka-2.8/
* io/opentelemetry/javaagent/instrumentation/spring/kafka/v2_7 -> io/opentelemetry/javaagent/instrumentation/spring/kafka/v2_8

Reply:
That version number is based on the earliest supported version. Even if we compile agains 2.8 the instrumentation still works for 2.7 that we use to run tests.